src/shared/mock/fees.ts

```diff
@@ -1,11 +0,0 @@
-import { assetAmount } from '@xchainjs/xchain-util'
```

Comment:
?

Reply:
It is all BNB based. 
I can change it to BTC

Reply:
> ?


/**
 * BNB fees
 **/
export const BNB_TRANSFER_FEES: TransferFees = {
  single: assetAmount(0.000375),
  multi: assetAmount(0.0003)
}

This hardcoded fee for BNB. This is no longer required.